src/modeler/AutoRest.Swagger/Model/ValidationUtilities.cs

```diff
@@ -262,20 +262,23 @@ public static bool IsArrayTypeResponseOperation(Operation op, ServiceDefinition
             if (op.Responses["200"]?.Schema?.Reference?.Equals(string.Empty) == false)
             {
                 var modelLink = op.Responses["200"].Schema.Reference;
+
```

Comment:
Not that I know of, go ahead and file an issue suggesting this as a linter rule and someone should get to it.